src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.view
+
+class InputView {
```

Comment:
```suggestion
object InputView {
```

We can make the view classes as objects.
What do you think are the differences between a class and an object?
Official Kotlin documentation can be a great source to find this out :)

Reply:
Thanks for your nice suggestion.
I think class is used to create multiple instances with different states. One must instantiate it manually. An object is created as a single instance automatically and used for utilities or shared logic.
Since I’m using `InputView` and `OutputView` without needing multiple instances, defining them as `object` is better.